templates/repo/commits_table.tmpl

```diff
@@ -29,8 +29,8 @@
 	</div>
 {{end}}
 
-{{if and .Commits (gt .CommitCount 0)}}
-	{{template "repo/commits_list" .}}
```

Comment:
Where is still using the `repo/commits_list`? Why not unify them to the new layout?

Reply:
![image](https://github.com/user-attachments/assets/5b8040a2-4a71-4045-9cfb-cdb2db31569c)
`repo/wiki/revision`

Reply:
Why not unify them to the new layout?

Reply:
The old logic is as follows. I noticed that the revision page of the GitHub wiki also uses a table layout, so I'm not sure if we should change our wiki to a timeline layout.